### PR TITLE
eamt-2022-ingestion

### LINKED ISSUE #1883: Ingestion request EAMT 2022
Hello

We are preparing the proceedings for the EAMT 2022 conference. We found on [this website](https://aclanthology.org/info/contrib/) that we should start an issue here to get the proceedings in the anthology. 

Repository that we are tracking the proceedings in [this repo](https://github.com/ardate/EAMT2022-proceedings/) (WIP, based on previous edition of EAMT 2020). [Meta data](https://github.com/ardate/EAMT2022-proceedings/blob/master/ACLPUB/info/meta.txt) (correct). So the venue name is `eamt` and there is only one volume (main conference), as title `Proceedings of the 23nd Annual Conference of the European Association for Machine Translation`.

Conference starts June 1st. We would like to negotiate 29th of May as a publication date.

Do you need anything else from us?

### PR DESCRIPTION
**Pull request by xinru1414**
closes #1883

### CODE REVIEW

File: data/xml/2022.eamt.xml [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+      <url hash="d735fdf8">2022.eamt-1.5</url>
+      <bibkey>loock-etal-2022-use</bibkey>
+    </paper>
+    <paper id="6">
+      <title>Comparing and combining tagging with different decoding algorithms for back-translation in <fixed-case>NMT</fixed-case>: learnings from a low resource scenario</title>
+      <author><first>Xabier</first><last>Soto</last></author>
+      <author><first>Olatz Perez-De-Vi
+
+textasciitilde</first><last>Naspre</last></author>
+      <author><first>Gorka</first><last>Labaka</last></author>
```
Comment (mjpost) [inline data/xml/2022.eamt.xml:79]: This is obviously mis-parsed (should be Perez-De-Viñaspre). @xinru1414 what does the source look like? Can you fix this in the ingestion script? Same for `\"{i}`.

File: data/xml/2022.eamt.xml [hunk 2]
```diff
@@ -0,0 +1,881 @@
...
+      <author><first>Evgeny</first><last>Matusov</last></author>
+      <pages>349–350</pages>
+      <abstract>Video dubbing is the activity of revoicing a video while offering a viewing experience equivalent to the original video. The revoicing usually comes with a changed script, mostly in a different language, and the revoicing should reproduce the original emotions, coherent with the body language, and lip synchronized. In this project, we aim to build an AD system in three phases: (1) voice-over; (2) emotional voice-over; (3) full dubbing, while enhancing the system with human-in-the-loop capabilities for a higher quality.</abstract>
+      <url hash="ce1ff18c">2022.eamt-1.65</url>
+      <bibkey>gangi-etal-2022-automatic</bibkey>
+    </paper>
+    <paper id="66">
+      <title>Overview of the <fixed-case>ELE</fixed-case> Project</title>
+      <author><first>“Itziar</first><last>Aldabe</last></author>
+      <author><first>Jane</first><last>Dunne</last></author>
```
Comment (mjpost) [inline data/xml/2022.eamt.xml:795]: weird stray quote here? was this in the source?

Comment (BramVanroy) [reply data/xml/2022.eamt.xml:795]: FYI I think this may have been an issue from copying from/to Google Sheets. Apologies for that!

**Comment by BramVanroy** [comment]
Thanks a lot for your work!

**Pull request merged**